App Engine Standard should always use the uberjar (#21634)
quarkusio/quarkus: docs/src/main/asciidoc/deploying-to-google-cloud.adoc

This will create a default service for your App Engine application.

> App Engine Standard cannot understand Quarkus' specific packaging layout, therefore, you must set up your application to be packaged as an uber-jar via your `application.properties` file:

Reviewer: ```suggestion
App Engine Standard does not support the default Quarkus' specific packaging layout, therefore, you must set up your application to be packaged as an uber-jar via your `application.properties` file:
```

[source, properties]
----
quarkus.package.type=uber-jar
----

> Then, you can choose to build the application by yourself or letting `gcloud` or the Google Cloud Maven plugin build it for you.

Reviewer: ```suggestion
Then, you can choose to build the application manually or delegating that responsibility to `gcloud` or the Google Cloud Maven plugin.
```
